README.md

```diff
@@ -78,3 +78,19 @@ A VSCode debug config is available to run and debug the project.
 
 To make the exporter talk with a Ceph RGW S3 endpoint, create a copy of the `.env.example` file and name it `.env`.
 Be sure ot add your Ceph RGW S3 endpoint and credentials in it.
+
+### Create a new Release
+
+To create a new release (example is for release `v0.1.2`):
+
+1. Increase the version according to Semantic Versioning in the [`VERSION` file](VERSION).
+2. Add a new entry to the [`CHANGELOG.md`](CHANGELOG.md) with the changes and improvements listed in it.
+3. Set the new version, which will be the new container image tag, in [the `values.yaml` of the Helm chart here](https://github.com/koor-tech/extended-ceph-exporter/blob/main/charts/extended-ceph-exporter/Chart.yaml#L24) (`appVersion:` field).
```

Comment:
Does changing this require also creating a new release on helm? Also, why not sync the version numbers in [the helm chart](charts/extended-ceph-exporter/Chart.yaml) so that we're not maintaining two versioning systems for the same project? @galexrt

Reply:
A Helm chart can be changed without the software, it is used for deploying, around it being changed (e.g., adjusting a Helm template to use a newer K8S API, adding a new functionality to create new objects).